EtherealEngine/etherealengine
# Use inputs to click in editor

## Summary
Clicking was broken when moving to ECS everything

## References
closes #_insert number here_

## QA Steps
Check every component that needs to be moved by transforms. Ensure they all have inputComponents. For the sake of 1.6 I am putting this up for review and merge now. This allows most things to be clicked
